### PR TITLE
Cb 17

### PR DESCRIPTION
**Pull request by mrabiabrn**
Added swagger extension and documented view and edit post API functionalities

### CODE REVIEW

**Review by kocasariumut — DISMISSED** [review]
works as intended

**Review by hamzaakyildiz — APPROVED** [review]

**Review by CodeBlo — APPROVED** [review]

**Review by asi345 — DISMISSED** [review]
The unit tests may have some import error.

**Review by asi345 — DISMISSED** [review]
Required changes are done correctly but I had problems running the Swagger. It might be a syntax error in 'apiDocs' reference in your code. You can change it to 'apidocs'.

**Review by kocasariumut — DISMISSED** [review]
works as intended

**Review by hamzaakyildiz — APPROVED** [review]
Seems good

**Review by ramazanbulut — APPROVED** [review]
seems to be mergeable

File: practice-app/backend/api/viewPost/viewPost.py [hunk 1]
```diff
@@ -15,15 +16,19 @@
     the post's tags. Returns this with other information. 
 '''
 @viewPostDetails_bp.route('/api/viewPost/<int:postId>', methods=['GET'])
+@swag_from('../../apiDocs/viewPost/viewPost.yml')
```
Comment (mrabiabrn) [inline practice-app/backend/api/viewPost/viewPost.py:19]:
```suggestion
@swag_from('../../apidocs/viewPost/viewPost.yml')
```

File: practice-app/backend/api/editPost/editPost.py [hunk 2]
```diff
@@ -21,26 +21,27 @@
     message and HTTP 200 code. If not, returns HTTP 403 error indicating that the user is not 
     authorized to perform this action.
 '''
-@editPostDetails_bp.route('/api/editPost/<string:username>/<int:postId>', methods=['POST'])
-def editPost(username, postId):
+@editPostDetails_bp.route('/api/editPost/<string:ownerUsername>/<int:postId>', methods=['POST'])
+@swag_from('../../apiDocs/editPost/editPost.yml')
```
Comment (mrabiabrn) [inline practice-app/backend/api/editPost/editPost.py:25]:
```suggestion
@swag_from('../../apidocs/editPost/editPost.yml')
```

**Review by kocasariumut — APPROVED** [review]
works as intended

**Review by erencanuysal — APPROVED** [review]

**Review by asi345 — APPROVED** [review]
Required changes are done correctly.

**Pull request merged**